client/src/components/InlinePanel/index.js

```diff
@@ -208,6 +208,36 @@ export class InlinePanel extends ExpandingFormset {
       },
     );
   }
+  
+  /** 
+    * Add tabindex -1 into newly created form if attr not present and
+      remove attr from old forms on blur event, if not present previously,
+  */
+  initialFocus(prefix) {
+    const forms = this.formsElt.children(':not(.deleted)');
+    const inlineChildId = 'inline_child_' + prefix;
+    const formsArray = forms.toArray();
+
+    formsArray.forEach((node) => {
+      const $node = $(node);
+
+      if ($node.attr('id') === inlineChildId) {
+        if (!$node.attr('tabindex')) {
+          $node.attr('tabindex', -1);
+          $node[0].scrollIntoView({ behavior: 'smooth' });
+          $node.focus();
+
+          // blur event listener to remove tabindex when it loses focus
+          $node.one('blur', () => {
+            if ($node.attr('tabindex') === '-1') {
+              $node.removeAttr('tabindex');
+            }
+          });
+        }
+      }
+    });
+  }
+
 
```

Comment:
This is a good start but as per comment below, we are doing a lot of 'work' here that we do not need to do.

1. No need to go through each form that is not disabled, instead we can just pass in a selected node via known ID structure.
2. Also, the code here is now NOT scrolling if tabIndex is already set, this is not what we want to do, we want to always scroll/focus but only set/remove the tabIndex if not already on the element.
3. Finally, the JSDoc needs a bit of formatting work and should include a note about the focusing now.

```suggestion  /**
   * Add tabindex -1 into newly created form if attr not present and
   * remove attr from old forms on blur event, if not present previously.
   * Always scroll and then focus on the element.
   */
  initialFocus($node) {
    if (!$node || !$node.length) return;

    if (!$node.attr('tabindex')) {
      $node.attr('tabindex', -1);
      // blur event listener to remove tabindex when it loses focus
      $node.one('blur', () => {
        if ($node.attr('tabindex') === '-1') {
          $node.removeAttr('tabindex');
        }
      });
    }

    $node[0].scrollIntoView({ behavior: 'smooth' });
    $node.focus();
  }
```